modules/contents.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable brace-style */
```

Comment:
- [ ] [OPTIONAL] Kindly try to avoid ignoring the linters, instead if there is an error you should fix it